scripts/run.sh

```diff
@@ -129,6 +134,49 @@ RECO_DIR=RECO/${TAG}
 RECO_TEMP=${TMPDIR}/${RECO_DIR}
 mkdir -p ${RECO_TEMP} ${BASEDIR}/${RECO_DIR}
 
+register_to_rucio(){
+
+python <<EOF
+from rucio.client import Client
+from rucio.common import exception
+from rucio.common.utils import adler32
+
+parent_dir="$1"
```

Comment:
Avoid the here-doc since it easily leads to incorrect substitutions, and it mixes languages. Can we just have a .py script in this directory?

Reply:
Alright. I will do that.